### PR TITLE
Added documentation for setting up DNS Server

### PR DESCRIPTION
**Pull request by daperoni**
Information on starting a DNS server through Azure Portal.

### CODE REVIEW

File: tests/docs/DNS-Setup.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Setting up DNS for CCF Network
```
Comment (jazzybluesea) [inline tests/docs/DNS-Setup.md]: Please preview and make sure the images are aligned to the left border.

File: tests/docs/DNS-Setup.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## VM Configuration
```
Comment (jazzybluesea) [inline tests/docs/DNS-Setup.md]: Please make sure any confidential information, like the subscription id is crossed out in all the images.

File: tests/docs/DNS-Setup.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+## Preparing network
+### First VM
+
+The first node starts the network and acts as the primary. In a directory, create symbolic links to the cchost and enclave files. Copy the gov.lua file to this location. 
+
+```
+
+ln -s <path-to-cchost> cchost
+ln -s <path-to-enclave-file> <enclave-file-name>
+cp <path-to-gov.lua> ./gov.lua
```
Comment (jazzybluesea) [inline tests/docs/DNS-Setup.md]: Don't you also have to copy the network_cert.pem file?

Comment (daperoni) [reply tests/docs/DNS-Setup.md]: The networkcert.pem file is generated by the first node.

File: tests/docs/DNS-Setup.md (hunk 2 above)
Comment (daperoni) [reply tests/docs/DNS-Setup.md]: I believe this has been fixed. Do you see anything I missed?

File: tests/docs/CCF-NetworkSetup.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
+
+
+
+# Starting a CCF network
+## Simple Test Netwrk
```
Comment (jazzybluesea) [inline tests/docs/CCF-NetworkSetup.md]: Network

File: tests/docs/CCF-NetworkSetup.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+First build CCF and the cloud sealing application. To start a network, call
```
Comment (jazzybluesea) [inline tests/docs/CCF-NetworkSetup.md]: It should say "test network"

**Comment by jazzybluesea** [comment]
LGTM

**Pull request merged**